addons/http_routing/models/ir_http.py

```diff
@@ -389,7 +378,8 @@ def _add_dispatch_parameters(cls, func):
         if request.routing_iteration == 1:
             context = dict(request.context)
             path = request.httprequest.path.split('/')
-            is_a_bot = cls.is_a_bot()
+            user_agent = request.httprequest.user_agent.string.lower()
+            is_a_bot = is_a_bot(user_agent)
```

Comment:
the variable `is_a_bot` will overwrite the function `is_a_bot`, can you do `mics.is_a_bot` instead ?